components/var_dumper/introduction.rst

```diff
@@ -72,6 +72,13 @@ current PHP SAPI:
        to your ``php.ini`` file;
     #. From time to time, run ``composer global update`` to have the latest
        bug fixes.
+       
+The dump_die() Function
+-------------------
+
+When you do not want to execute any other code right after dumping a variable: 
+
+``dump_die($var);``
```

Comment:
Exactly, think extra helpers will not be implemented. Otherwise I'll make
another docs PR

Op wo 11 feb. 2015 23:08 schreef Luis Cordova <notifications@github.com>:

> In components/var_dumper/introduction.rst
> <https://github.com/symfony/symfony-docs/pull/4988#discussion_r24538600>:
>
> > @@ -72,6 +72,13 @@ current PHP SAPI:
> >         to your ``php.ini`` file;
> >      #. From time to time, run ``composer global update`` to have the latest
> >         bug fixes.
> > +
> > +The dump_die() Function
> > +-------------------
> > +
> > +When you do not want to execute any other code right after dumping a variable:
> > +
> > +``dump_die($var);``
>
> @xabbuh <https://github.com/xabbuh> i think @fabpot
> <https://github.com/fabpot> rejected this PR head on don't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony/symfony-docs/pull/4988/files#r24538600>.
>